package.json

```diff
@@ -29,7 +29,9 @@
     "locale": "lerna run locale",
     "audit": "lerna exec -- npm audit",
     "audit:fix": "lerna exec -- npm audit fix",
-    "test": "lerna run test --stream -- -- --passWithNoTests --coverage",
+    "test": "lerna run test --stream -- -- --passWithNoTests --coverage --coverageDirectory=.coverage && npm run coverage:merge && npm run coverage:report",
+    "coverage:merge": "npx istanbul-merge --out .nyc_output/coverage.json packages/api/.coverage/coverage-final.json packages/api-react/.coverage/coverage-final.json packages/core/.coverage/coverage-final.json packages/gui/.coverage/coverage-final.json  packages/icons/.coverage/coverage-final.json packages/wallets/.coverage/coverage-final.json",
+    "coverage:report": "npx nyc report --reporter=html --reporter=text --reporter=lcov",
```

Comment:
`--reporter=lcov` is `--reporter=lcov-only` + `--reporter=html`.
So `--reporter=html` isn't necessary unless you want to have the report files under `.coverage/` directory as well as `.coverage/lcov-report/`. (The same html files would be genereated to both directory)

Reply:
Thanks, I'll make that change. This was a GitHub Copilot suggestion :)